packages/mobile/src/verify/saga.test.ts

```diff
@@ -0,0 +1,685 @@
+      isVerified: false,
+    })
+    const mockKomenciContextWithUnverifiedMtwAddress = {
+      ...mockKomenciContext,
+      unverifiedMtwAddress: mockAccount1,
+      sessionActive: true,
+    }
+    const mockGetActionableAttestations = jest.fn()
+    mockGetActionableAttestations.mockReturnValue([mockActionableAttestation])
+
+    await reduxSagaTestPlan
+      .expectSaga(fetchOnChainDataSaga)
+      .provide([
+        [select(e164NumberSelector), mockE164Number],
+        [call(getContractKit), contractKit],
+        [select(komenciContextSelector), mockKomenciContextWithUnverifiedMtwAddress],
+        [select(overrideWithoutVerificationSelector), true],
+        [select(shouldUseKomenciSelector), true],
+        [select(phoneHashSelector), mockPhoneHash],
+        [
+          call([contractKit.contracts, contractKit.contracts.getAttestations]),
+          mockAttestationsWrapper,
+        ],
+        {
+          call: ({ fn }, next) =>
+            fn === getActionableAttestations ? mockGetActionableAttestations() : next(),
+        },
+      ])
+      .put(setActionableAttestation([mockActionableAttestation]))
+      .put(setOverrideWithoutVerification(undefined))
+      .put(
+        setVerificationStatus({
+          isVerified: false,
+        })
+      )
+      .put(doVerificationFlow(true))
+      .run()
+  })
+
+  it('succeeds with Komenci disabled', async () => {
+    const contractKit = await getContractKitAsync()
+    ;(mockAttestationsWrapper.getVerifiedStatus as jest.Mock).mockReturnValue({
+      isVerified: false,
+    })
+    const mockKomenciContextWithUnverifiedMtwAddress = {
+      ...mockKomenciContext,
+      unverifiedMtwAddress: mockAccount1,
+      sessionActive: true,
+    }
+    const mockGetActionableAttestations = jest.fn()
+    mockGetActionableAttestations.mockReturnValue([mockActionableAttestation])
+
+    await reduxSagaTestPlan
+      .expectSaga(fetchOnChainDataSaga)
+      .provide([
+        [select(e164NumberSelector), mockE164Number],
+        [call(getContractKit), contractKit],
+        [select(komenciContextSelector), mockKomenciContextWithUnverifiedMtwAddress],
+        [select(overrideWithoutVerificationSelector), true],
+        [select(shouldUseKomenciSelector), false],
+        [select(phoneHashSelector), mockPhoneHash],
+        [call(getAccount), mockAccount],
+        [
+          call([contractKit.contracts, contractKit.contracts.getAttestations]),
+          mockAttestationsWrapper,
+        ],
+        {
+          call: ({ fn }, next) =>
+            fn === getActionableAttestations ? mockGetActionableAttestations() : next(),
+        },
+      ])
+      .put(setActionableAttestation([mockActionableAttestation]))
+      .put(setOverrideWithoutVerification(undefined))
+      .put(
+        setVerificationStatus({
+          isVerified: false,
+        })
+      )
+      .put(doVerificationFlow(true))
+      .run()
+  })
+  it('fails', async () => {
```

Comment:
we should provide a longer description, something like `shows an error when an error is thrown` so it's easier to understand the test when looking at it.
This feedback is for other places too (and other files) where a similar short description was used.